src/manifestoo/main.py

```diff
@@ -168,6 +188,14 @@ def callback(
     # addons selection
     if select_addons_dirs:
         main_options.addons_selection.add_addons_dirs(select_addons_dirs)
+    if select_git_modified:
+        modified_addons = get_branch_modified_addons(
+            select_git_modified, main_options.addons_path
```

Comment:
I'm not sure using the addons path here is good, as the addons path might be set by ODOO_RC, and point to several git clones. So the result might end up confusing.

To avoid an additional option to specify the git directory, we could make this look in the current directory.

Reply:
I understand the point, since you need to be somewhere in the git tree to run a command with a git option.
However, for each modified file, you have to know if it's from and Odoo module or 'something else'. So a file modified in `odoo/addons/odoo_keycloak/` is in an odoo addon, but not one in `doc/service/keycloak`. How do you know the difference? It's supposed to be set in the addons path, and I don't see any other way to know. Just checking the local directory is a bit painful if you want next to have different subfolders to look into, and it's already a known usage pattern, so that seems like the right way to do it to me.

I understand the idea of "looking into the current folder" but it's not even how we work currently.
For now the existing commands we have on the project are all at the root of the git project and pass `odoo/addons` as a prefix. It would be very awkward to add `cd` commands before/after, and in the case of multiple directories to then stitch the results together.